uploadModel.py

```diff
@@ -40,7 +40,7 @@
 upload_file_to_minio(jwt, image_url, image_path)
 
 print("Start Model ingest")
-if parent_model == None:
+if parent_model == None or parent_model == "":
```

Comment:
Could probably just be 
```suggestion
if not parent_model:
```